wave-api/src/test/groovy/io/seqera/wave/api/ContainerConfigTest.groovy

```diff
@@ -51,50 +51,50 @@ class ContainerConfigTest extends Specification {
         def c0 = new ContainerConfig()
         and:
         def l1 = new ContainerLayer( 'http://foo.com', 'sha256:12345', 100, 'sha256:67890' )
-        def c1 = new ContainerConfig(['/entry/point.sh'], ['/my/cmd'], ['FOO=1'], '/work/dir', [l1])
+        def c1 = new ContainerConfig(['/entry/point.sh'], ['/my/cmd'], ['FOO=1'], '/work/dir', [l1], [foo:'bar'])
 
         expect:
-        c0.toString() == 'ContainerConfig[entrypoint=null; cmd=null; env=null; workingDir=null; layers=[]]'
-        c1.toString() == 'ContainerConfig[entrypoint=[/entry/point.sh]; cmd=[/my/cmd]; env=[FOO=1]; workingDir=/work/dir; layers=[ContainerLayer[location=http://foo.com; tarDigest=sha256:67890; gzipDigest=sha256:12345; gzipSize=100]]]'
+        c0.toString() == 'ContainerConfig[entrypoint=null; cmd=null; env=null; workingDir=null; layers=[]; labels=null]'
+        c1.toString() == 'ContainerConfig[entrypoint=[/entry/point.sh]; cmd=[/my/cmd]; env=[FOO=1]; workingDir=/work/dir; layers=[ContainerLayer[location=http://foo.com; tarDigest=sha256:67890; gzipDigest=sha256:12345; gzipSize=100]]; labels={foo=bar}]'
     }
 
     def 'should validate empty' () {
         expect:
         new ContainerConfig().empty()
-        new ContainerConfig([], null, null, null, null).empty()
-        new ContainerConfig(null, [], null, null, null).empty()
-        new ContainerConfig(null, null, [], null, null).empty()
-        new ContainerConfig(null, null, null, '', null).empty()
-        new ContainerConfig(null, null, null, null, []).empty()
+        new ContainerConfig([], null, null, null, null, null).empty()
+        new ContainerConfig(null, [], null, null, null, null).empty()
+        new ContainerConfig(null, null, [], null, null, null).empty()
+        new ContainerConfig(null, null, null, '', null, null).empty()
+        new ContainerConfig(null, null, null, null, [], null).empty()
         and:
-        !new ContainerConfig(['x'], null, null, null, null).empty()
-        !new ContainerConfig(null, ['x'], null, null, null).empty()
-        !new ContainerConfig(null, null, ['x'], null, null).empty()
-        !new ContainerConfig(null, null, null, 'x', null).empty()
-        !new ContainerConfig(null, null, null, null, [new ContainerLayer()]).empty()
+        !new ContainerConfig(['x'], null, null, null, null, null).empty()
+        !new ContainerConfig(null, ['x'], null, null, null, null).empty()
+        !new ContainerConfig(null, null, ['x'], null, null, null).empty()
+        !new ContainerConfig(null, null, null, 'x', null, null).empty()
+        !new ContainerConfig(null, null, null, null, [new ContainerLayer()], [:]).empty()
```

Comment:
I would break in 2 distinct lines, keeping the cases of `layers` and `labels` distinct, e.g.:
```
!new ContainerConfig(null, null, null, null, [new ContainerLayer()], null).empty()
!new ContainerConfig(null, null, null, null, null, Map.of('k', 'v')).empty()
```

Reply:
done